docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -231,3 +169,23 @@ A number of advanced topics that were discussed here have been moved out, see be
 === User managed cache
 
 See <<usermanaged.adoc#,the user managed cache documentation>> for more information on this feature.
+
+=== Byte-sized heap
+
+See <<tiering.adoc#byte-sized-heap,the tiering documentation>> for more information on this feature.
+
+=== Update ResourcePools
+
+See <<tiering.adoc#update-resourcepools,the tiering documentation>> for more information on this feature.
+
+=== Off-heap
+
+See <<tiering.adoc#off-heap,the tiering documentation>> for more information on this feature.
+
+=== Disk persistence
+
+See <<tiering.adoc#disk-persistence,the tiering documentation>> for more information on this feature.
+
+=== Three tiers
```

Comment:
I do not feel that one is needed, I would instead but the section id somewhere inside the storage tiers section. A good place would be the `include` of the three tiers example.